src/game/common/thing/thingtemplate.cpp

```diff
@@ -493,13 +493,13 @@ void ThingTemplate::Set_Copied_From_Default()
 
 void ThingTemplate::Resolve_Names()
 {
-    for (unsigned int i = 0; i < m_prerequisites.size(); i++) {
-        m_prerequisites[i].Resolve_Names();
+    for (ProductionPrerequisite &rerequisite : m_prerequisites) {
+        rerequisite.Resolve_Names();
     }
 
-    for (unsigned int i = 0; i < m_prerequisites.size(); i++) {
+    for (ProductionPrerequisite &rerequisite : m_prerequisites) {
```

Comment:
this goes for the other loop in this function as well `**p**rerequisite`

Reply:
Ok.